### ISSUE TITLE
default acl - bug or bad documentation

### ISSUE THREAD

**Issue by hloeffler** [ansible/ansible#13290]
Issue Type:    Bug Report |  Documentation Report
Ansible Version:  ansible 2.0.0 (devel)

Documentation ( http://docs.ansible.com/ansible/acl_module.html ) says i can use "defaul:user:$user:$acl" but with 2.0 i cant use it anymore.

So, fix Documentation or code?

```
  - hosts: localhost
  tasks:
    - acl: name=/tmp entry="default:user:root:rw-" state=present
```

Expected Results:

```
OK
```

Actual Results:

```
TASK [acl entry=default:user:root:rw- state=present name=/tmp] *****************
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "'entry' MUST have 3 sections divided by ':' when 'state=present'."}
```

**Comment by hloeffler** [comment]
ansible 2.0 will break playbooks with this

**Comment by chrrrles** [comment]
CC'ing @astorije and @bcoca

**Comment by hloeffler** [comment]
reopen in "ansible/ansible-modules-core" ?
this is v2 prio because it breaks playbooks

**Comment by abadger** [comment]
Closing this since the ansible-modules-core ticket is open.  We'll be looking at that ASAP.

**Comment by cadavre** [comment]
When this fix will be available in prod? I'm using 2.0.0.2 (by ppa:) and this error still occurs.

``` yml
    - name: Set acl
      acl:
        path: "/some/var"
        entry: "default:u:dauser:rwX"
        recursive: yes
        state: present
```

Causes:

```
fatal: [dev]: FAILED! => {"changed": false, "failed": true, "msg": "'entry' MUST have 3 or 4 sections divided by ':' when 'state=present'."}
```
